virttest/qemu_devices/qcontainer.py

```diff
@@ -1018,6 +1023,27 @@ def machine_q35(cmd=False):
                 devices.append(qdevices.QStringDevice('fdc',
                                                       child_bus=qdevices.QOldFloppyBus('floppy'))
                                )
+
+            # add default pcie root port plugging pcie device
+            port_name = '%s-0' % root_port_type
+            port_params = {
+                'type': root_port_type,
+                # reserve slot 0x0 for plugging in  pci bridge
+                'reserved_slots': '0x0'}
```

Comment:
Is there some bug against the address auto allocation? otherwise, `bus.insert` could handle this well and we don't have to do it manually.

Reply:
pci-bridge must be in slot 0x  else device under plug into the bridge will not be scan in guest.